### DISCUSSION TITLE
FTP Adapter Not Found

### DISCUSSION THREAD

**Discussion by temwa** [spatie/laravel-backup#1691] [Q&A] [answered]
Hi,

When I run the clean or backup artisan commands I get he error below. Please assist. What am I getting wrong?

```
forge@webmobyle2:~/go.webmobyle.com$ php artisan backup:clean
Starting cleanup...

In FilesystemManager.php line 202:

Class "League\Flysystem\Ftp\FtpAdapter" not found
```

**Comment by temwa** [comment]
Maybe I have asked in the wrong section since there is no response. I am going to ask again as a bug.

**Reply by parallels999** [reply]
> I am going to ask again as a bug.

That is not necessary

**Comment by parallels999** [comment, marked as answer]
> Class "League\Flysystem\Ftp\FtpAdapter" not found

it seems you forgot to install the package

**Reply by temwa** [reply]
Thanks. I had tried to install composer `league/flysystem-ftp` but it did not work. After your response I tried and it worked. I guess it did not take the first time for some reason.

Thank you once again.
